core/domain/app_feedback_report_jobs_one_off_test.py

```diff
@@ -0,0 +1,242 @@
+                ticket_id='%s.%s.%s' % (
+                    'random_hash', self.TICKET_CREATION_TIMESTAMP.second,
+                    '16CharString1234'),
+                submitted_on=self.REPORT_SUBMITTED_TIMESTAMP,
+                report_type=self.REPORT_TYPE_SUGGESTION,
+                category=self.CATEGORY_OTHER,
+                platform_version=self.PLATFORM_VERSION,
+                device_country_locale_code=(
+                    self.DEVICE_COUNTRY_LOCALE_CODE_INDIA),
+                android_device_model=self.ANDROID_DEVICE_MODEL,
+                android_sdk_version=self.ANDROID_SDK_VERSION,
+                entry_point=self.ENTRY_POINT_NAVIGATION_DRAWER,
+                text_language_code=self.TEXT_LANGUAGE_CODE_ENGLISH,
+                audio_language_code=self.AUDIO_LANGUAGE_CODE_ENGLISH,
+                web_report_info=self.WEB_REPORT_INFO,
+                web_report_info_schema_version=(
+                    self.WEB_REPORT_INFO_SCHEMA_VERSION)))
+        expiring_web_report_model.created_on = self.TIMESTAMP_OVER_90_DAYS
+        expiring_web_report_model.put()
+
+    def test_scrubs_no_models(self):
+        self._add_current_reports()
+
+        job_id = (
+            app_feedback_report_jobs_one_off.ScrubReportsOneOffJob.create_new())
+        app_feedback_report_jobs_one_off.ScrubReportsOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_mapreduce_tasks()
+
+        output = (
+            app_feedback_report_jobs_one_off.ScrubReportsOneOffJob.get_output(
+                job_id))
+        self.assertEqual(output, [])
+
+        current_model = (
+            app_feedback_report_models.AppFeedbackReportModel.get_by_id(
+                self.REPORT_ID_AT_90_DAYS))
+        self.assertFalse(current_model is None)
+        self.assertTrue(current_model.scrubbed_by is None)
+
+    def test_scrubs_all_models(self):
+        self._add_expiring_reports()
+
+        job_id = (
+            app_feedback_report_jobs_one_off.ScrubReportsOneOffJob.create_new())
+        app_feedback_report_jobs_one_off.ScrubReportsOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_mapreduce_tasks()
+
+        output = (
+            app_feedback_report_jobs_one_off.ScrubReportsOneOffJob.get_output(
+                job_id))
+        self.assertEqual(output, [])
+
+        android_model = (
+            app_feedback_report_models.AppFeedbackReportModel.get_by_id(
+                self.ANDROID_REPORT_ID_OVER_90_DAYS))
+        self.assertFalse(android_model is None)
+        self.assertEqual(
+            android_model.scrubbed_by, feconf.REPORT_SCRUBBER_BOT_ID)
+
+        web_model = (
+            app_feedback_report_models.AppFeedbackReportModel.get_by_id(
+                self.WEB_REPORT_ID_OVER_90_DAYS))
+        self.assertFalse(web_model is None)
+        self.assertEqual(web_model.scrubbed_by, feconf.REPORT_SCRUBBER_BOT_ID)
+
+    def test_standard_operation(self):
```

Comment:
This test name is a bit unclear. Maybe something like: ``test_job_expired_and_new_reports_only_old_expired`` or similar (might be able to be shortened further).